### PR TITLE
Beginner guide touch up

### PR DESCRIPTION
**Pull request by ante185**
Organised the vanilla factions, added the new vanilla factions
Added missing RHS, 3CB, CUP factions to the list
Added pulau under single maps
Added Mortain to map list and code
Added IFA factions, modsets

### CODE REVIEW

File: source/beginners_guide/raw_beginners_guide.rst [hunk 1]
```diff
@@ -234,11 +260,12 @@ Supported Modsets & Maps
       Single maps with their own required mods:
 
       - `Anizay <https://steamcommunity.com/sharedfiles/filedetails/?id=1537973181>`_
+      - `UMB Columbia <https://steamcommunity.com/sharedfiles/filedetails/?id=2266710560>`_
```
Comment (targetingsnake) [inline source/beginners_guide/raw_beginners_guide.rst:263]: this was alphabeticly sorted, `UMB Columbia` starts with a 'u' so it should be further down the line

File: source/beginners_guide/raw_beginners_guide.rst [hunk 2]
```diff
@@ -430,6 +457,9 @@ How to set up a game
          *  - Normandy
             - :code:`Antistasi_SPE_Normandy.SPE_Normandy`
 
+         *  - Mortain
```
Comment (targetingsnake) [inline source/beginners_guide/raw_beginners_guide.rst:460]: please put this before Normandy, then we would have a good alphabetic order

File: source/beginners_guide/raw_beginners_guide.rst [hunk 3]
```diff
@@ -218,6 +243,7 @@ Supported Modsets & Maps
       Available maps with the `Spearhead 1944 CDLC <https://store.steampowered.com/app/1175380/Arma_3_Creator_DLC_Spearhead_1944/>`_:
 
       - Normandy
+      - Mortain
```
Comment (targetingsnake) [inline source/beginners_guide/raw_beginners_guide.rst:246]: switch Normandy and mortain for alphabetic order pls

**Review by targetingsnake — CHANGES_REQUESTED** [review]
looks good, please check for the correct alphabetic order as the lists where ordered by it

**Comment by targetingsnake** [comment]
@ante185 please recheck i did the ordering real quick would merge tomorrow afternoon/evening

**Review by targetingsnake — APPROVED** [review]

**Comment by Giddius** [comment]
**__Just as fyi:__**

If there is a problem with building it or something else I can help, then don't hesitate to reach out to me *(dm, ping, carrier pigeon)*!

**Pull request merged**